modules/core/packet-server/keeper/keeper.go

```diff
@@ -8,14 +8,14 @@ import (
 
 type Keeper struct {
 	cdc           codec.BinaryCodec
-	channelKeeper types.ChannelKeeper
-	clientKeeper  types.ClientKeeper
+	ChannelKeeper types.ChannelKeeper
```

Comment:
made this public so they were retrievable in testchains through the packetserver.

We already do this with IBCKeeper